src/views/well-architected-framework/collection-view/index.tsx

```diff
@@ -44,6 +44,7 @@ export default function WellArchitectedFrameworkCollectionView({
 				isOrdered={ordered}
 				tutorials={tutorials.map((t: ClientTutorialLite) => ({
 					id: t.id,
+					collectionId: id,
```

Comment:
`collectionId` is now needed for `TutorialCard` (or rather `TutorialCardWithAuth` as used in `TutorialCardGrid`), in order to display progress.